packages/wallets/src/features/cashier/modules/WithdrawalVerification/WithdrawalVerificationRequest/WithdrawalVerificationRequest.tsx

```diff
@@ -9,24 +11,26 @@ type TProps = {
 const WithdrawalVerificationRequest: React.FC<TProps> = ({ sendEmail }) => {
     return (
         <div className='wallets-withdrawal-verification-request'>
-            <div className='wallets-withdrawal-verification-request-icon'>
-                <EmailVerification />
-            </div>
-            <div className='wallets-withdrawal-verification-request__messages'>
-                <p className='wallets-withdrawal-verification-request__title'>
-                    Please help us verify your withdrawal request.
-                </p>
-                <p className='wallets-withdrawal-verification-request__description'>
-                    <span>
+            <WalletsActionScreen
+                actionText='Send email'
+                // TODO: Add line breaks
```

Comment:
I believe we can use the <WalletText/> component here instead of <p/>

Reply:
which component @nijil-deriv ?